source/switching_to_live/set_up_a_live_worldpay_account/index.html.md.erb

```diff
@@ -1,187 +1,95 @@
 ---
-title: Switching to live
-weight: 100
+title: Connect your live account to Worldpay
+weight: 200
 ---
 
-# Set up a live Worldpay account
+# Connect your live account to Worldpay
 
-Please read the guidance on what to do [before you switch to
-live](/switching_to_live/before_you_switch_to_live/#before-you-switch-to-live)
-first.
+Before you start, you must:
 
-Worldpay uses the term “production” for live accounts.
+- read about [what to do first](/switching_to_live/#before-you-start)
+- [request a live GOV.UK Pay account](/switching_to_live/#1-request-a-live-gov-uk-pay-account)
 
-You should follow these instructions in order.
+You must also be an admin on your GOV.UK Pay account.
 
-You can also read about setting up live
-[ePDQ](/switching_to_live/set_up_a_live_epdq_account) and
-[Smartpay](/switching_to_live/set_up_a_live_smartpay_account) accounts.
+## Check your Worldpay account
 
-## Prerequisites
+Make sure your Worldpay account is both:
 
-Before you begin, make sure you have an admin account with both Worldpay and
-GOV.UK Pay. You need an account with the Worldpay Corporate Gateway. GOV.UK
-Pay does not support the Worldpay Business Gateway.
+- an admin account
+- using the Worldpay Corporate Gateway - not the Worldpay Business Gateway
 
-## Set up your merchant code
+## Switch to Production Mode for your merchant code
 
 Sign in to your [Worldpay
-account](https://secure.worldpay.com/sso/public/auth/login.html) and use
-__Select MerchantCode__ to find your merchant code.
+account](https://secure.worldpay.com/sso/public/auth/login.html) and select your merchant code.
 
-If __Switch to Production__ is at the bottom of the left-hand menu, toggle it on.
+If __Test Mode__ is at the bottom of the left-hand menu, switch to __Production Mode__. Worldpay uses the term 'production' for live accounts.
 
-## Set up your profile
+## Copy your details into GOV.UK Pay
```

Comment:
This heading seems to narrow for the content of this section, which involves lots more than just copying details into Pay.